client/knode/validator/validator.go

```
@@ -301,6 +301,8 @@ func (val *validator) AddVote(vote *types.Vote) error {
 	}
 
 	if err := val.votingSystem.Add(addressVote); err != nil {
+		//fixme: we ignore all errors, e.g. duplicate vote
+		log.Error("cannot add the vote", "err", err)
```

Theres a story for this, maybe we should just return the err for the moment